common/api/core-common.api.md

```diff
@@ -5533,6 +5547,13 @@ export interface Localization {
     unregisterNamespace(namespace: string): void;
 }
 
+// @public
+export enum LockLevel {

```

Comment:
core-backend has a `LockState` with the same semantics you are capturing here. Move it to core-common?

Reply:
I am not sure we need `LockLevel` here at all. `LockLevel` is defined in iModels clients as a string enum because iModels API returns lock level as a string. Platform already has `LockState` to convey the same information about a lock and `BackendHubAccess` implementation already has code to [convert the string lock level to `LockState`](https://github.com/iTwin/imodels-clients/blob/3c87539effae754c2da3e959e0494c9b3cb838ec/itwin-platform-access/imodels-access-backend/src/interface-adapters/ClientToPlatformAdapter.ts#L63). We can and should do the same conversion when adapting the [error that iModels API client throws](https://github.com/iTwin/imodels-clients/blob/4a3d178a8acc3228e889171e67f5937f4cf3b7a8/clients/imodels-client-authoring/src/base/types/apiEntities/LockErrorInterfaces.ts#L18).

Reply:
Removed LockLevel and moved LockState to core-common from core-backend